packages/x-tailwindcss/demo/src/components/xds-button.vue

```diff
@@ -76,10 +82,14 @@
         Layout: this.layouts.map(addParentClasses(this.base)),
         Outline: this.colors.map(addParentClasses(this.base, this.outline)),
         Ghost: this.colors.map(addParentClasses(this.base, this.ghost)),
+        Tight: this.colors.map(addParentClasses(this.base, this.tight)),
+        Link: this.colors.map(addParentClasses(this.base, this.link)),
         Disabled: [
           this.base,
           addParentClasses(this.base)(this.outline),
-          addParentClasses(this.base)(this.ghost)
+          addParentClasses(this.base)(this.ghost),
+          addParentClasses(this.base)(this.tight),
+          addParentClasses(this.base)(this.link)
```

Comment:
What do you think of adding combinations including the new variants?

Reply:
done